src/test/regress/expected/multi_mx_transaction_recovery.out

```diff
@@ -50,15 +50,29 @@ PREPARE TRANSACTION 'citus_12_should_commit';
 BEGIN;
 CREATE TABLE should_be_sorted_into_middle (value int);
 PREPARE TRANSACTION 'citus_12_should_be_sorted_into_middle';
+-- this node (e.g., node id 12) should not touch
+-- transactions with different nodeIds in the gid
+BEGIN;
+CREATE TABLE table_should_do_nothing (value int);
+PREPARE TRANSACTION 'citus_122_should_do_nothing';
 -- Add "fake" pg_dist_transaction records and run recovery
 INSERT INTO pg_dist_transaction VALUES (12, 'citus_12_should_commit');
 INSERT INTO pg_dist_transaction VALUES (12, 'citus_12_should_be_forgotten');
+INSERT INTO pg_dist_transaction VALUES (122, 'citus_122_should_do_nothing');
 SELECT recover_prepared_transactions();
  recover_prepared_transactions 
 -------------------------------
                              3
 (1 row)
 
+-- delete the citus_122_should_do_nothing transaction
+DELETE FROM pg_dist_transaction RETURNING *;
```

Comment:
is it worth adding a filter to make it less dependent on other tests?

Reply:
It is probably hard to run this test in parallel with others and also having prepared txs in `pg_dist_transaction` before the test starts. But still, I added the filter.